src/frequenz/sdk/timeseries/ev_charger_pool/_ev_charger_pool.py

```diff
@@ -53,7 +53,26 @@ class EVChargerData:
 
 
 class EVChargerPool:
-    """Interactions with EV Chargers."""
+    """An interface for interaction with pools of EV Chargers.
+
+    !!! note
+        `EVChargerPool` instances are not meant to be created directly by users. Use the
+        [`microgrid.ev_charger_pool`][frequenz.sdk.microgrid.ev_charger_pool] method for
+        creating an instance.
```

Comment:
Optionally to be consistent:
```suggestion
        creating `EVChargerPool` instances.
```